README.md

```diff
@@ -43,16 +43,15 @@ for more detailed information about the library.
 
 ### Submitting a Bug Report
 
-Swift Markdown tracks all bug reports with [Swift JIRA](https://bugs.swift.org/).
+Swift Markdown tracks all bug reports with [GitHub Issues](https://github.com/apple/swift-markdown/issues).
 You can use the "Swift-Markdown" component for issues and feature requests specific to Swift Markdown.
 When you submit a bug report we ask that you follow the
 Swift [Bug Reporting](https://swift.org/contributing/#reporting-bugs) guidelines
 and provide as many details as possible.
 
 ### Submitting a Feature Request
 
-For feature requests, please feel free to create an issue
-on [Swift JIRA](https://bugs.swift.org/) with the `New Feature` type
+For feature requests, please feel free to file a [GitHub issue](https://github.com/apple/swift-markdown/issues/new?template=FEATURE_REQUEST.md)
```

Comment:
Currently, swift-markdown repo does not have issue templete.

```suggestion
For feature requests, please feel free to file a [GitHub issue](https://github.com/apple/swift-markdown/issues/new)
```

Reply:
Also would you mind using force push to commit only 1 commit?

Reply:
I have not cloned to the local repo so I don't think I can do it.  If any solution please mention :)